### PR TITLE
fix: reduce resource requests/limits for gitops

### PR DESCRIPTION
**Pull request by psturc**
related to https://github.com/redhat-appstudio/e2e-tests/pull/1077

### CODE REVIEW

File: components/gitops/development/core_service_resources_patch.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  namespace: gitops
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
```
Comment (psturc) [inline components/gitops/development/core_service_resources_patch.yaml:15]: Screenshot 2024-03-14 at 7 23 07

File: components/gitops/development/gitops-service-argocd.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+kind: ArgoCD
+metadata:
+  name: gitops-service-argocd
+  namespace: gitops-service-argocd
+spec:
+  repo:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 100Mi
```
Comment (psturc) [inline components/gitops/development/gitops-service-argocd.yaml:11]: Screenshot 2024-03-14 at 7 24 32

**Comment by psturc** [comment]
known issue KFLUXBUGS-580
/retest-required

**Review by jgwest — APPROVED** [review]

**Pull request merged**